include/exception/NotYetImplemented.hpp

```diff
@@ -0,0 +1,52 @@
+/***********************************************************************/
+/*  Copyright 2015 ToPS                                                */
+/*                                                                     */
+/*  This program is free software; you can redistribute it and/or      */
+/*  modify it under the terms of the GNU  General Public License as    */
+/*  published by the Free Software Foundation; either version 3 of     */
+/*  the License, or (at your option) any later version.                */
+/*                                                                     */
+/*  This program is distributed in the hope that it will be useful,    */
+/*  but WITHOUT ANY WARRANTY; without even the implied warranty of     */
+/*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the      */
+/*  GNU General Public License for more details.                       */
+/*                                                                     */
+/*  You should have received a copy of the GNU General Public License  */
+/*  along with this program; if not, write to the Free Software        */
+/*  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,         */
+/*  MA 02110-1301, USA.                                                */
+/***********************************************************************/
+
+#ifndef EXCEPTION_NOT_YET_IMPLEMENTED_
+#define EXCEPTION_NOT_YET_IMPLEMENTED_
+
+// Standard headers
+#include <string>
+#include <sstream>
+
+// ToPS headers
+#include "exception/Exception.hpp"
+
+namespace tops {
+namespace exception {
+
+/**
+ * @class NotYetImplemented
+ * @brief Report a not yet implemented exception
+ */
+class NotYetImplemented : public Exception {
+
+public:
+  NotYetImplemented(std::string method_name);
```

Comment:
Need a section separator comment `// Concrete methods`.

You can see all section separators (in the other to use them) inside `doc/GUARD.cpp`. The same structure presented by the file repeats itself in the `public`, `protected` and `private` sections of the class (which, in turn, must come in this order). For example:

```c++
class Blah : public Base {
 public:
  // Static variables
  const static int answer = 42;

  // Virtual methods
  virtual void print(std::ostream &os);

  // Concrete methods
  template<typename Num>
  Num sum(Num a, Num b);

 private:
  // Instance variables
  int first_num = 1;
  int second_num = 2;

  // Concrete methods
  void crazyPreprocessment();
};
```